mat/svd.go

```diff
@@ -208,19 +212,18 @@ func (svd *SVD) UTo(dst *Dense) *Dense {
 		capCols: c,
 	}
 	dst.Copy(tmp)
-
-	return dst
 }
 
 // VTo extracts the matrix V from the singular value decomposition. The first
 // min(m,n) columns are the right singular vectors and correspond to the singular
 // values as returned from SVD.Values.
 //
-// If dst is not nil, V is stored in-place into dst, and dst must have size
-// n×n if the full V was computed, size n×min(m,n) if the thin V was computed,
-// and VTo panics otherwise. If dst is nil, a new matrix of the appropriate size
-// is allocated and returned.
-func (svd *SVD) VTo(dst *Dense) *Dense {
+// If dst is empty, VTo will resize dst to be n×n if the full V was computed
+// and size n×min(m,n) if the thin V was computed. If dst is non-empty, then
+// VTo will panic if dst is not the appropriate size. VTo will also panic if
```

Comment:
"When dst is non-empty, VTo will panic if dst is not the appropriate size."

Reply:
Done.